src/main/java/uk/ac/ebi/ega/egafuse/runner/CommandLineOptionParser.java

```diff
@@ -62,6 +62,9 @@ public static CliConfigurationValues parser(OptionSet optionSet) throws IOExcept
                     .concat(" can't be used as mount point. Ensure that the directory path should exist."));
         }
         cliConfigurationValues.setMountPath(mntPath);
+        cliConfigurationValues
+                .setTreeStructureEnable(optionSet.valueOf("t").toString().equalsIgnoreCase("disable") ? false : true);
```

Comment:
Again this usage of the equals method is not null pointer exception safe

Reply:
Set the default value to enable in optionSet.valueOf("t") so NPE will never occurr